src/UCTNode.cpp

```diff
@@ -92,6 +92,7 @@ bool UCTNode::create_children(std::atomic<int>& nodecount,
 
     auto legal_sum = 0.0f;
     for (const auto& node : raw_netlist.first) {
+        if (node.first < 0.001f) continue;
```

Comment:
As a magic number, this should at the least be captured in a constant or, preferably, be made a configurable parameter (to be exposed by USE_TUNER).

Reply:
Agreed, but I'd like others to easily test this and for more to weigh in before making this more complicated than necessary. Here's a branch with more math to back it up: https://github.com/WhiteHalmos/leela-zero/commit/1d8ec2d8e0c11d3a57850a00dced1d1b5cba979c